static/app/views/replays/detail/console/useConsoleFilters.spec.tsx

```diff
@@ -78,29 +112,81 @@ const breadcrumbs = [
   },
 ];
 
-describe('replays/details/console', () => {
-  it('filters console breadcrumbs by log level', () => {
-    const result = filterBreadcrumbs(breadcrumbs, '', ['error', 'warning']);
-    expect(result.length).toEqual(2);
+describe('useConsoleFilters', () => {
+  beforeEach(() => {
+    mockBrowserHistoryPush.mockReset();
   });
 
-  it('filters console breadcrumbs by search term', () => {
-    const result = filterBreadcrumbs(breadcrumbs, 'comp', []);
-    expect(result.length).toEqual(2);
+  it('should update the url on user input', () => {
```

Comment:
this isn't really testing user input is it? it's more url params

Reply:
yeah.. that's very true. I'll rename it.


To test the user input properly we'd want to write an RTL test that loads up the component.